Assets/Scripts/Puzzles/Color/ICommandQueue.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace DN.Puzzle
+{
+	public interface ICommandQueue<T>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'out' keyword to parameter 'T' to make it 'covariant'.](https://app.codacy.com/manual/stepperman/pvb-programming/pullRequest?prid=5403921)